pages/auth/resetpassword.tsx

```diff
@@ -0,0 +1,341 @@
+import React, { ReactElement, useState } from "react";
+import { Formik } from "formik";
+import { Button, TextInput, Label, Alert, ErrorListItem, Description } from "@components/forms";
+import { useAuth } from "@lib/hooks";
+import { useTranslation } from "next-i18next";
+import { GetServerSideProps } from "next";
+import { Confirmation } from "@components/auth/Confirmation/Confirmation";
+import { serverSideTranslations } from "next-i18next/serverSideTranslations";
+import UserNavLayout from "@components/globals/layouts/UserNavLayout";
+import { checkOne } from "@lib/cache/flags";
+import Link from "next/link";
+import * as Yup from "yup";
+import { isLowerCase, isNumber, isSymbol, isUpperCase, isValidGovEmail } from "@lib/validation";
+import emailDomainList from "../../email.domains.json";
+
+const ResetPassword = () => {
+  const {
+    username,
+    cognitoError,
+    cognitoErrorDescription,
+    cognitoErrorCallToActionLink,
+    cognitoErrorCallToActionText,
+    cognitoErrorIsDismissible,
+    needsConfirmation,
+    setNeedsConfirmation,
+    resetCognitoErrorState,
+    sendForgotPassword,
+    confirmPasswordReset,
+  } = useAuth();
+
+  const { t } = useTranslation(["reset-password", "common"]);
+
+  // we don't put this state in useAuth since its very unique to this page only
+  const [initialCodeSent, setInitialCodeSent] = useState(false);
+
+  // validation schema for the initial form to send the forgot password verification code
+  const sendForgotPasswordValidationSchema = Yup.object().shape({
+    username: Yup.string()
+      .required(t("input-validation.required", { ns: "common" }))
+      .email(t("input-validation.email", { ns: "common" }))
+      .test(
+        "username-govEmail",
+        t("provideUsername.fields.username.error.validGovEmail"),
+        (value = "") => isValidGovEmail(value, emailDomainList.domains)
```

Comment:
When you will be rebasing on develop you will see that we do not need to pass any domains list anymore. The file has been removed so you will also have to remove the import at the top of the file

Reply:
Sounds good will pull changes now and make this change